legal/satzung.md

```diff
@@ -0,0 +1,122 @@
+# Satzung
+
+
+## Name, Sitz, Geschäftsjahr
+
+1.   Der Verein trägt den Namen *open energy modeling foundation* (oemof)
+2.   Sitz des Vereins ist $(SITZ).
+3.   Das Geschäftsjahr ist das Kalenderjahr.
+
+
+## Zweck und Ziele
+
+1. Der Verein verfolgt ausschließlich und unmittelbar gemeinnützige Zwecke im Sinne des Abschnitts „Steuerbegünstigte Zwecke“ der Abgabenordnung.
+2. Zweck des Vereins ist die Förderung von Wissenschaft und Forschung.
+    Dieser Zweck wird verwirklicht durch
+    1. Bereitstellung wissenschaftlicher Software unter OSI-anerkannten Open-Source-Lizenzen,
+    2. Unterstützung bei der Weiterentwicklung, Wartung und Dokumentation ebendieser Software, und
+    3. $(ZWECK)
+
+
+## Selbstlosigkeit
+
+1. Der Verein ist selbstlos tätig. Er verfolgt nicht in erster Linie eigenwirtschaftliche Zwecke.
+2. Die Mittel des Vereins dürfen nur für seine satzungsmäßigen Zwecke verwendet werden.
+    Die Mitglieder dürfen keine Gewinnanteile und in ihrer Eigenschaft als Mitglieder auch keine sonstigen Zuwendungen aus Mitteln des Vereins erhalten.
+3. Es darf keine Person durch Ausgaben, die dem Zweck des Vereins fremd sind, oder durch unverhältnismäßig hohe Vergünstigungen begünstigt werden.
+
+
+## Organe des Vereins
+
+1. Die Organe des Vereins sind
+    1. die Mitgliederversammlung und
+    2. der Vorstand.
+2. Der Vorstand besteht drei Mitgliedern,
+    1. der Sprecher*in,
+    2. der Schriftführer*in und
+    3. der Kassenwärt*in.
+3. Der Vorstand vertritt den Verein gerichtlich und außergerichtlich im Sinne des § 26 BGB,
+    wobei jedes Vorstandsmitglied für sich zur Vertetung des Vereins berechtigt ist.
+4. Der Vorstand gibt sich eine Geschäftsordnung, die von der Mitgliederversammlung bestätigt werden muss.
+
+
+## Treffen und Kommunikation
+
+1. Sämtliche Treffen des Vereins und seiner Organe können mittels Fernkommunikationsmittel unter Beachtung zumutbarer Identitätskontrollen durchgeführt werden.
+2. Benachrichtigungsfristen gelten als erfüllt, wenn die Benachrichtigung vor Verstreichen der Frist abgeschickt oder veröffentlicht wurde.
+3. Eine Benachrichtigung gilt als zugestellt, wenn sie an die letzte dem Verein bekannte Adresse verschickt oder auf der Webseite des Vereins veröffentlicht wurde wurde.
+4. Bei Zustimmung einer zu benachrichtigenden Person kann gegenüber dieser auf die Einhaltung von Benachrichtigungsfristen und -formalien verzichtet werden.
+
+
+## Mitgliedschaft
+
+1. Mitglied des Vereins kann jede volljährige natürliche Person werden.
```

Comment:
What did you decide about membership of institutions?

Reply:
At the moment, this is the proposal. It has not been decided yet.

The arguments so far are:

* keine minderjährige, da nicht voll geschäftsfähig --> gibt eventuell Probleme

natürliche Person. Begründung:
* zum einen da Institute  evtl. unterschiedlich leicht teilnehmen können
* zum anderen soll der Fokus auf den Personen liegen

Reply:
Ok danke. Ich würde da gern nochmal das Für und Wider diskutieren. Ich weiß, dass dem RLI, damals als wir die Abfrage zu den Statements der Institutionen gemacht hatten, wichtig war, dass Institutionen Mitglied werden können. Kann da aber nochmal nachhaken.

Reply:
Wann ist denn die nächste Besprechung? Ist es vielleicht sinnvoll, die Termine hier im GitHub anzukündigen? Oder macht ihr das schon?

Reply:
Thank you for the hint, this is a good point. There is a mailing list based on the people you wanted to help to elaborate the statute at the last oemof meeting. But this is quite a while ago ... there was one kick-off webco in May, then Patrik provided the draft of the statute, and then there was a second web-conferenc last monday, but it was only Patrik and me.

Since there is a concrete proposal now, it is probably time to start the discussion in a bigger round. Maybe lets further organize that organisational stuff in #60

Reply:
Note that the current draft allows institutions to join. They would be ordinary members just as persons.